### PR TITLE
[zh-cn] update <footer> element of HTML

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/web/html/element/footer/index.md [hunk 1]
```diff
@@ -1,37 +1,115 @@
...
+      <td>
+        任何接受<a href="/zh-CN/docs/Web/HTML/Content_categories#流式内容">流式内容</a>的元素。请注意，<code>&#x3C;footer></code> 元素不得是 {{HTMLElement("address")}}、{{HTMLElement("header")}} 或其他 <code>&#x3C;footer></code> 元素的后代。
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">隐式 ARIA 角色</th>
+      <td>
+        <a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role">contentinfo</a> 或
+        <a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/Generic_role">generic</a>
+       如果是 <a href="/zh-CN/docs/Web/HTML/Element/article">article</a>、<a href="/zh-CN/docs/Web/HTML/Element/aside">aside</a>、        <a href="/zh-CN/docs/Web/HTML/Element/main">main</a>、<a href="/zh-CN/docs/Web/HTML/Element/nav">nav</a> 或 <a href="/zh-CN/docs/Web/HTML/Element/section">section</a> 元素的后代，则为 <a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role">contentinfo</a> 或
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/footer/index.md:95]:
```suggestion
       如果是 <a href="/zh-CN/docs/Web/HTML/Element/article">article</a>、<a href="/zh-CN/docs/Web/HTML/Element/aside">aside</a>、<a href="/zh-CN/docs/Web/HTML/Element/main">main</a>、<a href="/zh-CN/docs/Web/HTML/Element/nav">nav</a> 或 <a href="/zh-CN/docs/Web/HTML/Element/section">section</a> 元素的后代，则为 <a href="/zh-CN/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role">contentinfo</a> 或
```

File: files/zh-cn/web/html/element/footer/index.md [hunk 2]
```diff
@@ -1,37 +1,113 @@
...
 
-- \<footer>元素内的作者信息应包含在{{HTMLElement("address")}} 元素中。
-- \<footer>元素不是章节内容，因此在[outline](/zh-CN/docs/Sections_and_Outlines_of_an_HTML5_document)中不能包含新的章节。
+- 在 {{HTMLElement("address")}} 元素中包含有关作者的信息，该元素可以包含在 `<footer>` 元素中。
+- 当最近的祖先分段内容或分段根元素是 body 元素时，页脚适用于整个页面。
+- `<footer>` 元素不是分节内容，因此不会在[大纲](/zh-CN/docs/Web/HTML/Element/Heading_Elements)中引入新的分节。
+
+## 示例
 
-## 例子
```
Comment (yin1999) [inline files/zh-cn/web/html/element/footer/index.md:26]:
```suggestion
```

File: files/zh-cn/web/html/element/footer/index.md [hunk 3]
```diff
@@ -1,37 +1,112 @@
...
+title: <footer>：页脚元素
 slug: Web/HTML/Element/footer
+l10n:
+  sourceCommit: fdd3ac5598c3ddceb71e59949b003936ae99f647
 ---
 
 {{HTMLSidebar}}
 
-**HTML \<footer> 元素**表示最近一个[章节内容](/zh-CN/docs/Web/Guide/HTML/Sections_and_Outlines_of_an_HTML5_document#Defining_Sections_in_HTML5)或者[根节点](/zh-CN/docs/Web/Guide/HTML/Sections_and_Outlines_of_an_HTML5_document#Sectioning_root)（sectioning root）元素的页脚。一个页脚通常包含该章节作者、版权数据或者与文档相关的链接等信息。
+[HTML](/zh-CN/docs/Web/HTML) **`<footer>`** 元素表示其最近的祖先[分段内容](/zh-CN/docs/Web/HTML/Content_categories#分段内容)的页脚或[分段内容](/zh-CN/docs/Web/HTML/Element/Heading_Elements#分段内容)元素。`<footer>` 通常包含有关该部分作者、版权数据或相关文档链接的信息。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/footer/index.md:10]: The [en-US document](https://github.com/mdn/content/blob/5671055d63552c5a4dc22ce7f6bea408afa1521a/files/en-us/web/html/element/footer/index.md?plain=1#L10) uses term "section root", and the link to target document is broken, please fix this.

File: files/zh-cn/web/html/element/footer/index.md [hunk 4]
```diff
@@ -1,37 +1,112 @@
...
-该元素仅包含 [全局属性](/zh-CN/docs/HTML/Global_attributes)。
+该元素仅包含[全局属性](/zh-CN/docs/Web/HTML/Global_attributes)。
 
 ## 使用说明
 
-- \<footer>元素内的作者信息应包含在{{HTMLElement("address")}} 元素中。
-- \<footer>元素不是章节内容，因此在[outline](/zh-CN/docs/Sections_and_Outlines_of_an_HTML5_document)中不能包含新的章节。
+- 在 {{HTMLElement("address")}} 元素中包含有关作者的信息，该元素可以包含在 `<footer>` 元素中。
+- 当最近的祖先分段内容或分段根元素是 body 元素时，页脚适用于整个页面。
+- `<footer>` 元素不是分节内容，因此不会在[大纲](/zh-CN/docs/Web/HTML/Element/Heading_Elements)中引入新的分节。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/footer/index.md:22]:
```suggestion
- `<footer>` 元素不是分段内容，因此不会在[大纲](/zh-CN/docs/Web/HTML/Element/Heading_Elements)中引入新的分段。
```

File: files/zh-cn/web/html/element/footer/index.md (hunk 3 above)
Comment (jasonren0403) [reply files/zh-cn/web/html/element/footer/index.md:10]:
```suggestion
[HTML](/zh-CN/docs/Web/HTML) **`<footer>`** 元素表示其最近的祖先[分段内容](/zh-CN/docs/Web/HTML/Content_categories#分段内容)的页脚或[分段根](/zh-CN/docs/Web/HTML/Element/Heading_Elements#标注章节内容)元素。`<footer>` 通常包含有关该部分作者、版权数据或相关文档链接的信息。
```

Comment (yin1999) [reply files/zh-cn/web/html/element/footer/index.md]: I mean: "section root" section is not existed in [Heading elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) 🤕

Comment (jasonren0403) [reply files/zh-cn/web/html/element/footer/index.md]: > I mean: "section root" section is not existed in [Heading elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) 🤕

Oops, I've raised https://github.com/mdn/content/pull/32817 to fix it

**Review by yin1999 — APPROVED** [review]
Thank you @jasonren0403

**Pull request merged**